app/src/main/java/org/jellyfin/androidtv/integration/dream/composable/DreamHost.kt

```diff
@@ -107,7 +107,20 @@ private suspend fun getRandomLibraryShowcase(
 			).drawable?.toBitmap()
 		} ?: return null
 
-		return DreamContent.LibraryShowcase(item, backdrop)
+		val logoUrl = api.imageApi.getItemImageUrl(
+			itemId = item.id,
+			imageType = ImageType.LOGO,
+			tag = tag,
+			format = ImageFormat.WEBP,
+		)
+
+		val logo = withContext(Dispatchers.IO) {
+			imageLoader.execute(
+				request = ImageRequest.Builder(context).data(logoUrl).build()
+			).drawable?.toBitmap()
+		} ?: return null
```

Comment:
We must support items without a logo and fallback to just showing the title like before. 

A lot of the items in my library do not have a logo set, so this would effectively make the screensaver useless.